src/site/components/phone.drupal.liquid

```diff
@@ -0,0 +1,19 @@
+{% for number in numbers %}
+{% assign numType = 'Phone' %}
+{% if number.entity.fieldPhoneNumberType == 'fax' %}
+{% assign numType = 'Fax' %}
+{% elseif number.entity.fieldPhoneNumberType == 'sms' %}
+{% assign numType = 'SMS' %}
+{% elseif number.entity.fieldPhoneNumberType == 'tty' %}
+{% assign numType = 'TTY' %}
```

Comment:
I think this is a great use case for the Liquid [case/when](https://shopify.github.io/liquid/tags/control-flow/) statement